packages/mdc-feature-targeting/_functions.scss

```diff
@@ -0,0 +1,170 @@
+@import "./variables";
```

Comment:
Missing `@license` in all of the files in `mdc-feature-targeting`.

Reply:
Good catch, I'll add those.